src/test/regress/multi_schedule

```diff
@@ -109,6 +109,7 @@ test: undistribute_table
 test: run_command_on_all_nodes
 test: background_task_queue_monitor
 test: other_databases
+test: grant_role_from_non_maindb
```

Comment:
Given that we're not doing any add/remove add op in this new test file, we should now be able to parallelize this with other_databases.

Reply:
Paralellized